docs/release-notes/NuGet-6.0.md

```diff
@@ -0,0 +1,387 @@
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+> [!NOTE]
+> Visual Studio 17.0, MSBuild 17.0, and .NET 6.0 require NuGet.exe 6.0 or later.
+
+## Summary: What's New in 6.0
+
+🎉 **This is the first release to offer full authoring and restoring support for NuGet packages targeting .NET 6.0** 🎉
+
+### .NET 6 Support
+
+NuGet 6.0 is the first release to offer full authoring and restoring support for NuGet packages targeting .NET 6.0. You can now target the following target frameworks:
+
+*   net6.0
+*   net6.0-windows
+*   net6.0-android
+*   net6.0-ios
+*   net6.0-macos
+*   net6.0-maccatalyst
+*   net6.0-tvos
+*   net6.0-tizen
+
+  ![.NET 6](media/NuGet-6.0/dotnet.png)
+
+If you aren’t familiar with the .NET 6.0 targets today or what it will look like in the future, don’t forget to check out the [.NET 6.0 TFM spec](https://github.com/dotnet/designs/blob/main/accepted/2021/net6.0-tfms/net6.0-tfms.md).
+
+### Source Mapping
+
+Earlier this year, many package managers became aware of [dependency confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) attacks in which a user to be tricked into installing a malicious dependency instead of the one they intended to. To fortify your software supply chain against these attacks, the NuGet team has developed a new feature that allows you to map your dependencies to specific sources. Below is an example of how you can use source mapping to protect your projects.
+
+<pre><code class="xml">&lt;!-- Define a global packages folder for your repository. --&gt;
+&lt;!-- This is where installed packages will be stored locally. --&gt;
+&lt;config&gt;
+  &lt;add key="globalPackagesFolder" value="globalPackagesFolder" /&gt;
+&lt;/config&gt;
+
+&lt;!-- Define my package sources, nuget.org and contoso.com. --&gt;
+&lt;!-- `clear` ensures no additional sources are inherited from another config file. --&gt;
+&lt;packageSources&gt;
+  &lt;clear /&gt;
+  &lt;!-- `key` can be any identifier for your source. --&gt;
+  &lt;add key="nuget.org" value="https://api.nuget.org/v3/index.json" /&gt;
+  &lt;add key="contoso.com" value="https://contoso.com/packages/" /&gt;
+&lt;/packageSources&gt;
+
+&lt;!-- Define mappings by adding package ID patterns beneath the target source. --&gt;
+&lt;!-- Contoso.* packages will be restored from contoso.com, everything else from nuget.org. --&gt;
+&lt;packageSourceMapping&gt;
+  &lt;!-- key value for &lt;packageSource&gt; should match key values from &lt;packageSources&gt; element --&gt;
+  &lt;packageSource key="nuget.org"&gt;
+    &lt;package pattern="*" /&gt;
+  &lt;/packageSource&gt;
+  &lt;packageSource key="contoso.com"&gt;
+    &lt;package pattern="Contoso.*" /&gt;
+  &lt;/packageSource&gt;
+&lt;/packageSourceMapping&gt;
+</code></pre>
+
+You can read more about source mapping in our [blog released earlier this year](https://devblogs.microsoft.com/nuget/introducing-package-source-mapping/).
```

Comment:
Link the local docs instead/as well?

Reply:
lol yes. I wrote these blurbs prior to merging your docs ;)